### PR TITLE
Attempt to fix a Docutils warning.

### PR DESCRIPTION
**Pull request by jy95**
"No MathJax URL specified, using local fallback"
This causes the debug mode of the ParsableText class to fail.

### CODE REVIEW

**Comment by jy95** [comment]
Example : 

```rst

    Reminder : Fibonacci's sequence is computed as follows

    .. math::

         f(n) =\begin{cases}0 & n=0 \\1 & n = 1\\ f(n-1) + f(n-2) & n > 1\end{cases}
```

File: inginious/frontend/parsable_text.py [hunk 1]
```diff
@@ -266,7 +266,7 @@ def rst(cls, string, show_everything=False, translation=gettext.NullTranslations
             'translation': translation,
             'raw_enabled': True,
             'file_insertion_enabled': False,
-            'math_output': 'MathJax'
+            'math_output': 'MathJax /this/does/not/need/to/exist.js'
```
Comment (anthonygego) [inline inginious/frontend/parsable_text.py]: Did you investigate to know what would be a real fix ?

Comment (jy95) [reply inginious/frontend/parsable_text.py]: As you can read from the [documentation](http://docutils.sourceforge.net/docs/user/config.html#math-output) : 
**"A URL pointing to a MathJax library should be appended after whitespace. A warning is given if this is missing."**
Feel free to use one of the three given solutions inside the documentation.

Comment (anthonygego) [reply inginious/frontend/parsable_text.py]: @GuillaumeDerval confirmed to me that this link is actually not used further. So I think it's OK for me.

**Pull request merged**